u_chess.py

```diff
@@ -0,0 +1,25 @@
+from chess_model import BoardView, BOARD_FACTORY, sprite_by_name
+
+def white_pawn_move(board, move):
+    if len(move) == 2:
+        if board[move] != ' ':
+            return 'illegal move'
+        rank = int(move[1])
+        file = move[0]
+        if rank == 4:
+            if board[move.replace(str(rank), str(rank - 1))] == 'w':
+                pawn_position = move.replace(str(rank), str(rank - 1))
+                return 'pawn moved to ' + move + ' from ' + pawn_position
```

Comment:
we can directly update the board here instead of returning a logger statement.
Edited: And return true for a successful board update